### ISSUE TITLE
Bundler 1.15.0 incompatibility

### ISSUE THREAD

**Issue by jonatanklosko** [rvm/pluginator#10]
A new version of bundler has been published a couple days ago and it causes some error.
There may be more, but this is the first that shows up.
```
jonatanklosko@mockingjay:~/projects/worldcubeassociation.org/WcaOnRails (master) $ bundle exec pre-commit run git
bundler: failed to load command: pre-commit (/home/jonatanklosko/.rbenv/versions/2.4.1/bin/pre-commit)
TypeError: no implicit conversion of Bundler::SpecSet into Array
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/autodetect/formatted_finder.rb:65:in `find_gem_specifications'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/autodetect/formatted_finder.rb:49:in `map_gem_plugins'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/autodetect/formatted_finder.rb:33:in `initialize'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/autodetect.rb:53:in `new'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/autodetect.rb:53:in `refresh'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/autodetect.rb:44:in `initialize'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator/extendable_autodetect.rb:42:in `initialize'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator.rb:33:in `new'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pluginator-1.4.0/lib/pluginator.rb:33:in `find'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pre-commit-0.33.0/lib/pre-commit/plugins_list.rb:4:in `pluginator'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pre-commit-0.33.0/lib/pre-commit/runner.rb:17:in `initialize'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pre-commit-0.33.0/lib/pre-commit/cli.rb:38:in `new'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pre-commit-0.33.0/lib/pre-commit/cli.rb:38:in `execute_run'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pre-commit-0.33.0/lib/pre-commit/cli.rb:21:in `execute'
  /home/jonatanklosko/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/pre-commit-0.33.0/bin/pre-commit:10:in `<top (required)>'
  /home/jonatanklosko/.rbenv/versions/2.4.1/bin/pre-commit:22:in `load'
  /home/jonatanklosko/.rbenv/versions/2.4.1/bin/pre-commit:22:in `<top (required)>'
```

**Comment by mpapis** [comment]
I will look into it in ~10h

**Comment by mpapis** [comment]
fixed in 1.4.1
